CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_zendesk v0.14.0
+
+## Bug Fixes
```

Comment:
should we note that this only affects people who use schedules in zendesk? might be implied bust just to be uber clear

Reply:
Yeah that's a good clarification to include.

Reply:
Updated.